mobile/src/main/java/ch/epfl/sdp/mobile/ui/setting/SettingScreen.kt

```diff
@@ -33,30 +34,37 @@ import ch.epfl.sdp.mobile.ui.profile.rememberSettingTabBarState
 fun SettingsScreen(
     state: SettingScreenState,
     modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(),
 ) {
   val lazyColumnState = rememberLazyListState()
   val tabBarState = rememberSettingTabBarState(state.pastGamesCount, state.puzzlesCount)
   val targetElevation = if (lazyColumnState.firstVisibleItemIndex >= 1) 4.dp else 0.dp
   val elevation by animateDpAsState(targetElevation)
 
-  UserScreen(
-      header = {
-        SettingHeader(
-            state = state,
-            modifier = Modifier.padding(vertical = 16.dp),
-        )
-      },
-      profileTabBar = {
-        SettingTabBar(
-            state = tabBarState,
-            modifier = Modifier.fillMaxWidth(),
-            elevation = elevation,
-        )
-      },
-      matches = state.matches,
-      onMatchClick = {},
-      lazyColumnState = lazyColumnState,
-      modifier = modifier)
+  Scaffold(
+      modifier = modifier,
+      content = { paddingValues ->
+        val totalPadding = paddingValues + contentPadding
+        UserScreen(
+            header = {
+              SettingHeader(
+                  state = state,
+                  modifier = Modifier.padding(vertical = 16.dp),
+              )
+            },
+            profileTabBar = {
+              SettingTabBar(
+                  state = tabBarState,
+                  modifier = Modifier.fillMaxWidth(),
+                  elevation = elevation,
+              )
+            },
+            matches = state.matches,
+            lazyColumnState = lazyColumnState,
+            modifier = modifier.fillMaxSize(),
+            contentPadding = totalPadding,
+            onMatchClick = {})
+      })

```

Comment:
What is this `Scaffold` used for ?

Reply:
Removed and replaced with UserScreen in e2d401e1c120ebfdd323d3d2971f29cb337d812b

Reply:
Removed and replaced with UserScreen in e2d401e1c120ebfdd323d3d2971f29cb337d812b